src/Troubleshooting/winehq.md

```diff
@@ -59,11 +60,12 @@ discord_rpc = false           # (X)
 [player.fflags]
 DFIntTaskSchedulerTargetFps = 60 # (X)
 ```
+
 Read over every option carefully and follow the instructions in the comments.
 
 **Critical**: This configuration file depends heavily on your Vinegar version! <u>Do **NOT** reuse it</u>, especially after updating Vinegar. Recheck this guide to ensure you have the latest version of the configuration.
 
-Once the file is ready, set the `WINEPREFIX` env var to a temporary location, then run Vinegar with the `-config (filepath)` flag to use the cfg you just created. 
+Once the file is ready, set the `WINEPREFIX` env var to a temporary location, then run Vinegar with the `-config (filepath)` flag to use the cfg you just created.
 
 The command should be similar to this if all steps were followed correctly: `WINEPREFIX=/tmp/report vinegar -config /home/jrelvas/Documents/winehq-report.toml player`.
```

Comment:
Why was this removed?

Reply:
Yeah, this shouldn't be removed. I even went out of my way to make it not outdated.

Reply:
It wasn't removed. GitHub doesn't show it properly. But I ran an auto formatter, and that's probably what triggered a change - such as changing the line ending.